ansible/roles/eos/templates/t2-core.j2

```diff
@@ -57,6 +57,15 @@ interface Management {{ mgmt_if_index }}
  ip address {{ mgmt_ip }}/{{ mgmt_prefixlen }}
  no shutdown
 !
+{% if enable_macsec is defined %}
```

Comment:
can we check here if the duthost we work on has the "macsec_Card" flag enabled in inventory. Else it will add macsec config to all ceos VM's irrespective of uplink/downlink linecards

Reply:
Removed the macsec config from t2-leaf.j2